internal/guest/bridge/bridge_v2.go

```diff
@@ -407,7 +407,7 @@ func (b *Bridge) waitOnProcessV2(r *Request) (_ RequestResponse, err error) {
 		}
 		exitCodeChan, doneChan = p.Wait()
 	} else {
-		c, err := b.hostState.GetContainer(request.ContainerID)
+		c, err := b.hostState.GetRunningContainer(request.ContainerID)
```

Comment:
Should we be able to wait on a container regardless of whether it has finished creating or not?

Reply:
I wanted to make sure that the affected code paths stay unchanged. I can make an issue to discuss what should be done for other operations later?

Reply:
works for me